### PR TITLE
Autocomplete

### PR DESCRIPTION
**Pull request by eduardavercosa**
## Objective and Motivation
- Users can see a list with the pokemon names as they type it

### CODE REVIEW

File: backend/pokemon/helpers.py [hunk 1]
```diff
@@ -29,6 +39,12 @@ def create_pokemon(pokemon_data):
     )
 
 
+def pokemon_in_api(poke_name):
+    url = POKE_API_URL + poke_name
+    response = requests.head(url)
+    return bool(response)
```
Comment (fjsj) [inline backend/pokemon/helpers.py:45]: Did you check if this is enough? Try with a non-existing pokemon.

**Review by fjsj — CHANGES_REQUESTED** [review]
Please check my last comment.

File: backend/pokemon/helpers.py (hunk 1 above)
Comment (eduardavercosa) [reply backend/pokemon/helpers.py:45]: I tried. If I type a pokemon that is not in the API, it says "ERROR: Choose only existing Pokemon."

**Review by fjsj — APPROVED** [review]
LGTM.

**Pull request merged**